parttwo.asm

```diff
@@ -0,0 +1,86 @@
+.data
+
+intro:	.asciiz "Hello. I am Nick Gordon. This program requests two integers.\nIt then does three calculations, and saves them to data.\nIt then prints those results.\n"
+outro:	.asciiz "This was the program! Hope you have a good day!"
+prompt1:	.asciiz "Please enter integer one: "
+prompt2:	.asciiz "Please enter integer two: "
+space:	.asciiz "\n" # because I have to use an entire word to store the newline :(
+sum:		.word	0
+sumstring:	.asciiz "Sum: "
+diff:		.word	0
+diffstring:	.asciiz "Diff: "
+prod:		.word	0
+prodstring: .asciiz "Prod: "
+
+.text
+
+la $s5, sum
+la $s6, diff
+la $s7, prod
```

Comment:
Loading all memory addresses into registers in the beginning is a bit inefficient because you can simply load them into the same register whenever you're ready to use them. OK though, no points lost.